provisioning/plugins/corim-psa-decoder/test-harness/build-test-vectors.sh

```diff
@@ -5,8 +5,30 @@
 set -eu
 set -o pipefail
 
+# function generate_go_test_vector constructs CBOR test vector using
+# supplied comid and corim json template and saves them in a file
+# $1 file name for comid json template
+# $2 file name for corim json template
+# $3 a qualifier for each cbor test vector name
+# $4 name of the file where the generated CBOR test vectors are aggregated
+generate_go_test_vector () {
+	echo "generating test vector using $1 $2"
+	cocli comid create -t ${1}.json
+	cocli corim create -m ${1}.cbor -t $2 -o corim${1}.cbor
+	echo "// automatically generated from $t.json" >> ${4}
+	echo "var ${3}${1} = "'`' >> ${4}
+	cat corim${1}.cbor | xxd -p >> ${4}
+	echo '`' >> ${4}
+}
+
+# CORIM TEMPLATE
 CORIM_TEMPLATE=corimMini.json
 
+# CORIM CCA TEMPLATES
+CORIM_CCA_TEMPLATE=corimCca.json
+CORIM_CCA_TEMPLATE1=corimCcaNoProfile.json
```

Comment:
rather than `TEMPLATE1` call it `TEMPLATE_NO_PROFILE`

Reply:
Done!